### DISCUSSION TITLE
Should the comment options be available under the posts?

### DISCUSSION THREAD

**Discussion by BartDubois** [origami-network/origami-network.github.io#8] [Q&A]
While there is value in the direct conversation under the certain post there is also a rumor what tool to use it.

I was considering using GitHub issues integration as the comments solution.
Unfortunately it has some limitations.

In the mine time the Discussions appears on the GitHub which looks like much better option.
There is no much examples how it can be embed into own page so it can take some time to do so if anyhow possible.

I will not mind that the requirement in both option is to have GitHib account.
While the blog is linked tight to software and management of it most of the readers should have there own account already.

**Comment by mrmattiago** [comment]
maybe this would be sufficient? https://github.com/hydecr/hyde-blog :)
it can be integrated with github out of the box
